src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+    
+    public boolean isMutating() { return false; }
```

Comment:
This is okay, but do consider if having isMutating default to returning false is intuitive enough for someone new to the codebase. This could open up the possibility of using an interface instead of inheritance.